OneSignalSDK/onesignal/src/main/java/com/onesignal/LocationGMS.java

```diff
@@ -85,51 +86,82 @@
       void complete(LocationPoint point);
    }
 
+   abstract static class LocationAndPromptHandler implements LocationHandler {
+      void onPromptHandle(boolean accepted) {}
+   }
+
    private static ConcurrentHashMap<CALLBACK_TYPE, LocationHandler> locationHandlers = new ConcurrentHashMap<>();
```

Comment:
Can we pleaseeeee change the name `CALLBACK_TYPE` to `PermissionType` or something. more relevant and not all capitol... Should be camel case, I know you didnt make this but it hurt my soul finding it...